lms/static/sass/xblocks/poll_survey.scss

```diff
@@ -124,6 +125,7 @@ body.new-theme {
             .poll-voting-thanks span {
                 background-color: #ffffff;
                 color: $black;
+                padding-left: 0em;
```

Comment:
why we need this?

Reply:
Ticket requirement, remove unwanted left spacing for Thank you note.

Reply:
ok thanks